config/routes.rb

```diff
@@ -1,4 +1,7 @@
 Rails.application.routes.draw do
+  devise_for :users
+  get '/cart', to: 'cart#show'
```

Comment:
Why do you guys have these 2 routes about the `cart` on this PR? Thought this story was about user sign in/up 🤔

Reply:
We merged with dev branch and put this implementation when we fixed merge conflicts. Guess we can still keep it as we gonna need it anyway later on, right?  Even though it's not related to that PR